wpninjas/ninja-forms
# fixes this issue that allows multi-part parts to be validated without…

… layouts and styles activated

Fixes #3376 

Related to this issue in multi-part:
https://github.com/wpninjas/ninja-forms-multi-part/issues/122

Changes proposed in this pull request:
- Added model.set( 'clean', false ) to field model when validating, this forces them to validate even if nothing has been changed( this is for multi-part part validation mainly )

To test:

- Create a multi-part form
- Turn on part validation
- Make sure some fields are required
- Make sure the Next button doesn't move you to the next part if there are errors on the page
- Make sure to try with and without Layouts & Styles activated.
- Make sure multi-part forms still work without 'part validation' turn on
- Make sure forms work with/without multi-part activated.

@wpninjas/developers
